### DISCUSSION TITLE
Whats the GraphQL Url for randomnet?

### DISCUSSION THREAD

**Discussion by memxor** [aptos-labs/aptos-developer-discussions#62] [Questions] [answered]
I believe it is `https://indexer-devnet.staging.gcp.aptosdev.com/v1/graphql` for devnet. But `https://indexer-random.staging.gcp.aptosdev.com/v1/graphql` or `https://indexer-randomnet.staging.gcp.aptosdev.com/v1/graphql` doesn't work.

**Comment by 0x-j** [comment, marked as answer]
Randomnet graphql is at `https://indexer-randomnet.hasura.app/v1/graphql`, we hard code these urls [here](https://github.com/aptos-labs/aptos-ts-sdk/blob/main/src/utils/apiEndpoints.ts#L8) in ts sdk in case you are curious.

**Reply by memxor** [reply]
Sorry I'm using Unity, the Aptos UnitySDK, its quite outdated.

**Reply by 0x-j** [reply]
Ah yes Unity SDK is very outdated, please don't use that
